FinalXmL/FinalXmL/Geonamedata.cs

```diff
@@ -2,64 +2,44 @@
 //
 // To parse this JSON data, add NuGet 'Newtonsoft.Json' then do:
 //
-//    using QuickType;
+//    using QuickTypeCityList;
 //
-//    var welcome = Welcome.FromJson(jsonString);
+//    var cityList = CityList.FromJson(jsonString);
```

Comment:
Remove this entirely.  It's considered bad practice to commit commented-out code to version control, because you can always go back in history to see what used to be there.